api/controller.py

```diff
@@ -496,6 +497,17 @@ def __call__(self):
         # their type, and redirect them to an appropriate feed.
         return self.appropriate_index_for_patron_type()
 
+    def authentication_document(self):
+        """Serve this library's Authentication For OPDS document."""
+        content = self.manager.auth.create_authentication_document()
```

Comment:
This could check if the document is already cached in self.manager.authentication_for_opds_documents, but it won't be called very often so I guess it doesn't matter much.